cpp/ql/lib/semmle/code/cpp/ir/implementation/raw/internal/TranslatedExpr.qll

```diff
@@ -3367,7 +3413,13 @@ class TranslatedVarArgsStart extends TranslatedNonConstantExpr {
     result = this.getVAList().getFirstInstruction(kind)
     or
     tag = VarArgsVAListStoreTag() and
-    result = this.getParent().getChildSuccessor(this, kind)
+    (
+      result = this.getParent().getChildSuccessor(this, kind)
+      or
+      // All store instructions could throw an SEH exception
+      result = this.getParent().getExceptionSuccessorInstruction(any(GotoEdge e), true) and
+      kind.(ExceptionEdge).isSEH()
```

Comment:
I don't think a call to `va_start` can ever generate a SEH exception, right 🤔?

Reply:
removed